src/main/Boards/HexNode.java

```diff
@@ -0,0 +1,40 @@
+//This class represents hex tiles as nodes with information about 
+//tile status (occupied by red, blue, etc.) and an ID which helps
+//locate a specific tile in the graph. It's based on the HexLocation
+//class of the Trevorah project.
+
+package main.Boards;
+
+public class HexNode {
+	
+	private int nodeID;
+	private int value;
+	//For red player, node value = 1; Blue player = 2.
+	
+	public HexNode(int nodeID) {
+		this.nodeID = nodeID;
+		this.value = 0;
+	}
```

Comment:
If value represents the color of the player (1, 2) why should the constructor without a value passed set the value to 0 (not a valid color)? As a group we should decide if the default color of the player should be red or blue in my opinion. Setting it to default as 0 doesn't make sense to me.